nss-cohort-36/bangazon-workforce-mgt-purely-magic
# Jl second ticket

* fixed departments list function to show number of employees working in that department
* completed department details to show department as a header and list employees in dept

to test:
commit your current branch
git fetch --all
git checkout jl-second-ticket

go to browser and localhost8000/departments 

ensure 4 departments are listed and employees are listed under each
select departments and see employees listed without duplicating department names
